libctru/source/console.c

```diff
@@ -187,7 +187,7 @@ ssize_t con_write(struct _reent *r,void *fd,const char *ptr, size_t len) {
 	int i, count = 0;
 	char *tmp = (char*)ptr;
 
-	if(!tmp || len<=0) return -1;
+	if(!tmp || len<0) return -1;
```

Comment:
`len<0` is always false since `len` is unsigned. That would simplify this to `if(!tmp) return -1`.